docs/services/frame-system/_index.md

```diff
@@ -11,6 +11,8 @@ tags: ["frame system", "services"]
 
 Any robot configured in Viam comes with the Frame System service: an internally managed and mostly static system for storing the "reference frame" of each component of a robot within a coordinate system configured by the user.
 
+![Visualization of a wheeled base configured with motors and a mounted camera in the Frame System tab of the Viam app UI](img/frame_system_wheeled_base.png)
+
```

Comment:
On line 21 - can you add where you click add frame? on the component panel I think?

Reply:
Yep!